### PR TITLE
New package: perl-Gtk2-Notify

### PR DESCRIPTION
**Pull request by chinarulezzz**

### CODE REVIEW

File: srcpkgs/perl-Gtk2-Notify/template [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+distfiles="${CPAN_SITE}/Gtk2/Gtk2-Notify-${version}.tar.gz"
+checksum=88189ae68dfbd54615ad133df07e2ec8048d06d8b9586add1227d74eb2ebb047
+wrksrc="Gtk2-Notify-${version}"
+
+do_build() {
+	perl Makefile.PL
+}
+
+pre_install() {
+	patch -p0 < ${FILESDIR}/libnotify.patch
```
Comment (Vaelatern) [inline srcpkgs/perl-Gtk2-Notify/template]: why is this not in `patches`?

File: srcpkgs/perl-Gtk2-Notify/template [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+depends="perl perl-Gtk2 libnotify"
+short_desc="Gtk2::Notify - Perl interface to libnotify"
+maintainer="chinarulezzz <s.alex08@mail.ru>"
+homepage="http://search.cpan.org/dist/Gtk2-Notify/"
+license="LGPL-2"
+distfiles="${CPAN_SITE}/Gtk2/Gtk2-Notify-${version}.tar.gz"
+checksum=88189ae68dfbd54615ad133df07e2ec8048d06d8b9586add1227d74eb2ebb047
+wrksrc="Gtk2-Notify-${version}"
+
+do_build() {
```
Comment (Vaelatern) [inline srcpkgs/perl-Gtk2-Notify/template]: Is this really the best way to do this? Is this really Module::Build underneath?

File: srcpkgs/perl-Gtk2-Notify/template [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Template build file for 'perl-Gtk2-Notify'
+pkgname=perl-Gtk2-Notify
+version=0.05
+revision=1
+build_style=perl-module
+hostmakedepends="perl perl-ExtUtils-Depends perl-ExtUtils-PkgConfig"
+makedepends="perl perl-Gtk2 gtk+-devel libnotify-devel"
+depends="perl perl-Gtk2 libnotify"
```
Comment (Vaelatern) [inline srcpkgs/perl-Gtk2-Notify/template]: `libnotify` redundant here
https://travis-ci.org/voidlinux/void-packages/jobs/186479933#L709-L714

File: srcpkgs/perl-Gtk2-Notify/template [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# Template build file for 'perl-Gtk2-Notify'
+pkgname=perl-Gtk2-Notify
+version=0.05
+revision=1
+build_style=perl-module
+hostmakedepends="perl perl-ExtUtils-Depends perl-ExtUtils-PkgConfig"
+makedepends="perl perl-Gtk2 gtk+-devel libnotify-devel"
```
Comment (Vaelatern) [inline srcpkgs/perl-Gtk2-Notify/template]: Is `perl` really a `makedepends`?

**Comment by chinarulezzz** [comment]
Fixed

**Pull request closed without merging**